x-pack/legacy/plugins/siem/public/components/ml_popover/upgrade_contents.tsx

```diff
@@ -38,14 +46,24 @@ export const UpgradeContents = React.memo(() => {
         />
       </EuiText>
       <EuiSpacer />
-      <EuiButton
-        href="https://www.elastic.co/subscriptions"
-        iconType="popout"
-        iconSide="right"
-        target="_blank"
-      >
-        {i18n.UPGRADE_BUTTON}
-      </EuiButton>
+      <EuiFlexGroup justifyContent="spaceBetween" gutterSize="none">
+        <EuiButton
+          href="https://www.elastic.co/subscriptions"
+          iconType="popout"
+          iconSide="right"
+          target="_blank"
+        >
+          {i18n.UPGRADE_BUTTON}
+        </EuiButton>
+        <EuiButton
+          href={`${chrome.getBasePath()}/app/kibana#/management/elasticsearch/license_management`}
+          iconType="popout"
+          iconSide="right"
+          target="_blank"
+        >
```

Comment:
You'll need to wrap each of the `EuiButton` components in a `EuiFlexItem` component. You'll also want to give each `EuiFlexItem` a prop of `grow={false}`.